csfieldguide/static/interactives/big-number-calculator/js/big-number-calculator.js

```diff
@@ -3,106 +3,110 @@
 const Big = require('big.js');
 
 $(document).ready(function () {
-    $('#interactive-big-number-calculator button').on('click', function(){
-        var button_type = $(this).attr('id').split('-').pop();
-        var x = getXValue();
-        var y = getYValue();
+  $('#interactive-big-number-calculator button').on('click', function(){
+    var button_type = $(this).attr('id').split('-').pop();
+    var x = getXValue();
+    var y = getYValue();
```

Comment:
Why the change from 4-space indents to 2-space? I thought 4 was the standard for JS?
Now that I look at it *almost* everything is 2-space, should I use that in my work?

Reply:
We follow Google's Javascript Style Guide which has 2-space indentation for JS. See https://google.github.io/styleguide/jsguide.html#formatting-block-indentation